docs/snippets/angular/app-story-with-mock.ts.mdx

```diff
@@ -0,0 +1,34 @@
+```ts
+// App.stories.ts
+
+import { Story, Meta } from '@storybook/angular/types-6-0';
+import { AppComponent } from './app.component';
+
+export default {
+  component: AppComponent,
+  title: 'App',
+} as Meta;
+
+const Template: Story<AppComponent> = (args: AppComponent) => ({
+  component: AppComponent,
```

Comment:
it is necessary to redefine the component here as it is already defined in `export default`

Reply:
When i adjust to 6.2 this and the other snippets will be adjusted accordingly. To reference only the component in the default export as mentioned throughout the comments. For now i'm good with it.